adobe/frl-online-proxy
# Address issues reported from field usage

## Summary
* This PR adds a /status endpoint to the running proxy.  GET requests to /status return a JSON body giving the proxy version and message that the proxy is running.  The returned status code is 200.
* This PR also cleans up the logging of requests to show the request verb, and not to log the (empty) body of a GET call.
* This PR also removes the debug logging of passwords.

## Testing Steps

For status endpoint and GET logging:
* Run the proxy at (say) `localhost:8080` or (say) `localhost:8443` (if you have a certificate). 
* Use the browser to navigate to `http://localhost:8080/status` or `https://localhost:8443/status`.
* You will see a status message similar to this one in your browser:
```{"message":"Proxy is running","proxyVersion":"1.1.0","statusCode":200}```
* Check that the logs do not include the (empty) body of the status request.

For password logging:
* Establish a config with both a cert password and a proxy password in the config file.
* Run the proxy.
* Check that both passwords are obscured in the log.
* Do the same test with no stored passwords, check that the config output is the same.

Fixes #34 
Fixes #21 
Fixes #29
